_build/maintainer_guide.rst

```diff
@@ -0,0 +1,332 @@
+Symfony Docs Maintainer Guide
+=============================
+
+The `symfony/symfony-docs`_ repository stores the Symfony project documentation
+and is managed by the `Symfony Docs team`_. This article explains in detail some
+of those management tasks, so it's only useful for maintainers and not regular
+readers or Symfony developers.
+
+Reviewing Pull Requests
+-----------------------
+
+All the recommendations of the `Symfony's respectful review comments`_ apply,
+but there are extra things to keep in mind for maintainers:
+
+* Always be nice in all interactions with all contributors.
+* Be extra-patient with new contributors (they show a special badge on GitHub).
+* Don't assume that contributors know what you think it's obvious to contribute
+  (e.g. lots of people don't know what to "squash commits" means).
+* Don't use acronyms like IMO, IIRC, etc. or complex English words (most
+  contributors are not native in English and it's intimidating for them).
+* Never engage in a heated discussion. Lock it right away using GitHub.
+* Never discuss non-tech issues. Some PRs are related to our Diversity initiative
+  and some people always try to drag you into politics. Never engage in that and
+  lock the issue/PR as off-topic on GitHub.
+
+Fixing Minor Issues Yourself
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It's common for new contributors to make lots of minor mistakes in the syntax
+of the RST format used in the docs. It's also common for non English speakers to
+make minor typos.
+
+Even if your intention is good, if you add lots of comments when reviewing a
```

Comment:
I have a general question regarding this paragraph. I agree with the sentiment, but I think adding a commit to someone else's pull request adds the risk for another point of frustration as it might feel like taking something away from them, giving the wrong impression of maintainers being in charge/taking control away from new contributors.

Maybe another approach would be to add a comment telling them that there are a few typos/minor issues that need to be addressed and offer to either fix them or point them out individually. I think this additional interaction, especially with first time contributors, might make them feel more welcome and included in the process and still offers them the opportunity to hand off when it feels to frustrating to implement lots of smaller changes.

Reply:
Thanks for your comment, but I'm going to keep the original comment. I still think that making 30 comments about very minor things in a PR sent by a new contributor is devastating for them.

However, thanks to your comment I've added the following, which is what we usually do:

```
If that person contributes again, it's OK to mention some of the minor
issues to educate them.
```